extensions/binding/org.eclipse.smarthome.binding.dmx/build.properties

```diff
@@ -0,0 +1,11 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/,\
+               lib/
+               lib/ola-java-client-0.0.3.jar,\
```

Comment:
Already commented on that in about.html. My fault: while most of OpenLighting is LGPL, the Java Client is Apache 2.0, and that is compatible.